main.py

```diff
@@ -30,46 +30,93 @@ def hello():
     return 'Hello, Circles!!'
 
 
-@app.route('/circles/api/v1.0/users/', defaults={'person_id': None})
-@app.route('/circles/api/v1.0/users/<int:person_id>', methods=['GET'])
-def get_person(person_id):
+@app.route('/circles/api/v1.0/users/<int:person_id>/', defaults={'resource': None})
+@app.route('/circles/api/v1.0/users/<int:person_id>/<resource>', methods=['GET'])
+def get_person(person_id, resource):
     # Request all users.
     if not person_id:
```

Comment:
I don't think `person_id` can be None anymore, so we can remove this section!

Reply:
Can't remove this-- the optional value is the resource being requested

absence of a resource will give the object itself (/jwei/) versus the presence of a resource will give something that is an edge away (/jwei/circles)